android-project/app/src/main/java/oceantreasur/es/fragments/WrongAnswerFragment.java

```diff
@@ -21,7 +23,7 @@ public int getIndicatorImage() {
 
     @Override
     public void playMusic() {
```

Comment:
This method should only return R.raw.superior_fart_sound. The logic should be in the base class.